client/src/rpc_client.rs

```diff
@@ -128,12 +128,32 @@ impl RpcClient {
         self.send_transaction_with_config(transaction, RpcSendTransactionConfig::default())
     }
 
+    fn cluster_tx_encoding(&self) -> Result<UiTransactionEncoding, RpcError> {
```

Comment:
```suggestion
    fn cluster_transaction_encoding(&self) -> Result<UiTransactionEncoding, RpcError> {
```

Reply:
Thanks!  I went ahead and squashed this change in (with updated call sites)